docs/getting-started/step-6-configure-add-ons.md

```diff
@@ -119,6 +119,71 @@ Resources:
       ...
 ```
 
+## aws-iam-authenticator
```

Comment:
@dominicgunn here's my first try at documenting this featuer
what do you tihnk?

Reply:
Sorry I took a while to get back to you. 

I've taken a glance and this is great & much better than our existing stuff. Thanks so much for the contribution.